lib-injection/sitecustomize.py

```diff
@@ -2,45 +2,168 @@
 This module when included on the PYTHONPATH will update the PYTHONPATH to point to a directory
 containing the ddtrace package compatible with the current Python version and platform.
 """
+
```

Comment:
Is it a bad idea to just wrap literally all the code in this file in a try except block to keep from ever crashing the application?

Reply:
I don't like having a `try/except` with so much code in it because it can become really tricky to figure out the origin of an exception that it catches. That said, I've put all of the nontrivial logic of sitecustomize into the `_inject` function and wrapped the call to that function in an exception handler.

Reply:
Ah I see, sorry missed how you did that. Great!